cvl/cvl_syntax.go

```diff
@@ -209,16 +209,19 @@ func (c *CVL) generateTableData(config bool, jsonNode *jsonquery.Node) (*yparser
 		}
 		if !topNodesAdded {
 			// Add top most conatiner e.g. 'container sonic-acl {...}'
-			topNode = c.yp.AddChildNode(modelInfo.tableInfo[tableName].module,
+			topNode, _ = c.yp.AddContainerNode(modelInfo.tableInfo[tableName].module,
```

Comment:
Why the 2nd output parameter introduced? Can the AddContainerNode() return any error? If yes, better not to ignore it. There is no nil check as well